plugin/pulp/plugin/profiler.py

```diff
@@ -0,0 +1,32 @@
+class Profiler(object):
```

Comment:
It's worth keeping an eye on this once we actually implement one, to see if it really deserves to be an entire class. As-is, it reminded me of this classic pycon talk. Although I recognize that this is really just an interface at the moment, so that logic may not apply.

https://youtu.be/o9pEzgHorH0?t=129

I think it's fine to re-visit this once a profiler actually gets implemented it, but I just wanted to share my thoughts. Another option would be to make it a static method on a unit model, which would also solve the discovery problem nicely.